src/main/java/org/influxdb/dto/Pong.java

```diff
@@ -25,6 +25,13 @@ public void setVersion(final String version) {
     this.version = version;
   }
 
+  /**
+   * Good or bad connection status.
+   */
+  public boolean isGood() {
+    return !"unknown".equalsIgnoreCase(version);
```

Comment:
@egoebelbecker would be possible to transform _"unknown"_ into a class scoped `static final` constant?

Reply:
Good point. I have no idea why I didn't do that in the first place. Cancelling this and submitting a new one.